packages/components/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -10,24 +10,18 @@ const meta: Meta<CheckboxProps> = {
   component: Checkbox,
   decorators: [
     (Story) => (
-      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+      <View
+        style={{
+          flex: 1,
+          justifyContent: 'center',
+          alignItems: 'center',
+          margin: 8,
+        }}>
```

Comment:
Thinking we might want to make this a default across all stories if possible

Reply:
Could do if you want to create a ticket. 

Could pull it out to the `storybook.tsx` utils file too however makes sense (just the style, the entire View to be a common story wrapper, whatever).